.claude/hooks/user-prompt-submit.sh

```
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# プロジェクトルート
+readonly project_root=$(
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)